app/src/main/java/org/kiwix/kiwixmobile/di/modules/ServiceModule.kt

```diff
@@ -1,5 +1,7 @@
 package org.kiwix.kiwixmobile.di.modules
 
+import android.app.Application
```

Comment:
Should we use `KiwixApplication` here?

Reply:
Nah, we are not depending on any specific functionality of KiwixApplication

Reply:
We just need to initialize the `WifiManager` instance. So, there's nothing we need from the modified `KiwixApplication` class.